extensions/adobe/experience/customerJourneyManagement/message-delivery-feedback.schema.json

```diff
@@ -37,7 +38,8 @@
             "bounce": "Message bounced",
             "delay": "Message delayed",
             "duplicate": "Duplicate message skipped",
-            "denylist": "Invalid destination skipped"
+            "denylist": "Invalid destination skipped",
+            "delivered": "Message Delivered"
```

Comment:
As **delivered** is not explicitly passed with other channels, we should update the description for the feedbackStatus that some enum values may not be applicable for all channels.

Reply:
updated